client/mds3/sampletable.js

```diff
@@ -367,7 +366,7 @@ export async function samples2columnsRows(samples, tk) {
 	const displayedFormatKeySet = new Set() // set of format keys for display, to skip keys not in display
 
 	// to be returned by this function, as inputs for renderTable
-	const columns = [{ label: 'Sample' }],
+	const columns = [{ label: tk.mds.termdbConfig.lollipop ? tk.mds.termdbConfig.lollipop.sample : 'Sample' }],
```

Comment:
improve as `label: arg.tk.mds.termdbConfig.lollipop?.sample || 'Sample'`

Reply:
ok it's done.